src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
-
+import "./App.css";
+import Footer from "./layouts/footer/Footer";
+import Header from "./layouts/header/Header";
 function App() {
```

Comment:
@youngjin-korea 

[제안]

import 한 코드라인과 비지니스 로직 간에 공백을 추가해주면 가독성이 더 좋을꺼 같아요.

```suggestion
import Header from "./layouts/header/Header";

function App() {
```